### PR TITLE
468 hosting provider page

### LINKED ISSUE #468: Create web hosting provider page with information that users can select from the drop down
See page design here:

https://xd.adobe.com/view/6934cd71-399f-43c2-40a2-1a41b966eb34-1f05/screen/d03670a5-7076-4958-86f2-218a97d828b7/AFTER-USER-TESTS-Desktop-Certbot-WebHostingProvide/

### PR DESCRIPTION
**Pull request by lschatzkin**
closes #468 

This page is for when user chooses Web Hosting Service for operating system, regardless of what is chosen under Software, per discussion with @okaysoraya . That is, if I understood correctly.

### CODE REVIEW

**Comment by lschatzkin** [comment]
I'm not understanding the test failure. I'm not sure how/where it is generating the failing links.  I don't see the links that are failing when I look at my sandbox.

**Comment by bmw** [comment]
I'm not 100% sure either. I think @esoterik may have a better idea as they wrote the code for the glossary earlier this year.

File: _scripts/instruction-widget/templates/getting-started/sharedhost.html [hunk 1]
```diff
@@ -1,13 +1,42 @@
...
+You’re probably not the only one using your web hosting provider service who wants HTTPS. You can request that your web hosting provider offer Let’s Encrypt HTTPS certificates as a free part of their hosting package. An effective way to make this ask is through email, their help desk system, or by contacting the web hosting provider through social media.</p>
+
+<p><span class="subhead">You can switch to a different web hosting provider.</span><br>
+Find a web hosting provider who offers full HTTPS support as part of their web hosting package by <a href="/hosting_providers">checking our list.</a></p>
+
+<p><span class="subhead">You might be able to use Certbot.</span><br>
+If you have <a href="/glossary/#ssh">SSH access</a> to the server your website is hosted on, you might be able to use Certbot. You will need to know the <a href="glossary/#software">software</a> and <a href="glossary/#operating-system">system</a> your server is running on. After you confirm the software and system information, you can use the dropdown menus above to generate specific instructions for running Certbot on your server through the command line.</p>
```
Comment (esoterik) [inline _scripts/instruction-widget/templates/getting-started/sharedhost.html]: Seems like the links without at leading "/" are failing the CI checks

Comment (lschatzkin) [reply _scripts/instruction-widget/templates/getting-started/sharedhost.html]: Thank you @esoterik so much for figuring this out! I managed to fix other broken links after this one, and now they are all fixed!

File: _scripts/instruction-widget/main.js [hunk 2]
```diff
@@ -139,6 +139,10 @@ $('document').ready(function() {
       selected = selected.split('-');
       $('#os-select').val(selected[0]);
       $('#server-select').val(selected[1].replace(".html", ""));
+      if (selected[0] == 'sharedhost') {
+        $('.use-certbot').hide();
```
Comment (wioux) [inline _scripts/instruction-widget/main.js]: :+1: :+1:

**Pull request merged**